evm_loader/cli/src/api_server/handlers/mod.rs

```diff
@@ -74,19 +73,6 @@ pub(crate) fn parse_emulation_params(
     (token, chain, max_steps, accounts, solana_accounts)
 }
 
-impl IntoResponse for NeonCliError {
```

Comment:
I think, we should make additional binary, for the API server, which will use neon_lib as dependency. So we will have both CLI and API server executables, each of them will work its own way (CLI for command line and API server for handling REST requests).

Reply:
The API server is a different binary already, check `cli/Cargo.toml`

Reply:
Yes, you are right, I had missed the thing that it is different binaries. But at this moment both of them are placed in the same crate. How about splitting them to separate crates and remove excessive dependencies?

Reply:
Ok, understood.